### PR TITLE
Finalizing the translations into Portuguese

### PR DESCRIPTION
**Pull request by vagnercruz**

### CODE REVIEW

File: lang/pt_BR.json [hunk 1]
```diff
@@ -295,15 +295,15 @@
...
     "Search games...": "Pesquisar jogos...",
-    "See more": "See more",
-    "Select a file": "Select a file",
+    "See more": "Ver mais",
+    "Select a file": "Selecione um arquivo",
     "Select a game": "Selecione um jogo",
     "Select an achievement": "Selecione uma conquista",
-    "Select an emulator": "Select an emulator",
-    "Select an issue": "Select an issue",
+    "Select an emulator": "Selevione um Emulador",
```
Comment (wescopeland) [inline lang/pt_BR.json:305]:
```suggestion
    "Select an emulator": "Selecione um Emulador",
```

File: lang/pt_BR.json [hunk 2]
```diff
@@ -110,27 +110,27 @@
...
     "Email addresses must match.": "Os endereços de e-mail devem corresponder.",
     "Email me": "Me envie um e-mail",
     "Email notifications": "Notificações por E-mail",
-    "Emulator": "Emulator",
-    "Emulator Core": "Emulator Core",
-    "Emulator Version": "Emulator Version",
-    "Example: The achievement requires collecting 10 coins. You collected 10 coins, but the achievement didn't unlock.": "Example: The achievement requires collecting 10 coins. You collected 10 coins, but the achievement didn't unlock.",
-    "Example: The achievement requires collecting 10 coins. You collected 5 coins, and it unlocked.": "Example: The achievement requires collecting 10 coins. You collected 5 coins, and it unlocked.",
+    "Emulator": "Emuladores",
+    "Emulator Core": "Núcleo do Emuladoro",
```
Comment (wescopeland) [inline lang/pt_BR.json:130]: possible typo: should this be "Emulador"?

File: lang/pt_BR.json [hunk 3]
```diff
@@ -312,19 +312,19 @@
     "Someone follows me": "Alguém me segue",
     "Something went wrong.": "Algo deu errado.",
     "Sort order": "Ordem de classificação",
-    "Started": "Started",
-    "Statistics": "Statistics",
+    "Started": "Inicado",
```
Comment (wescopeland) [inline lang/pt_BR.json:315]: possible typo: should this be "Iniciado"?

File: lang/pt_BR.json (hunk 2 above)
Comment (vagnercruz) [reply lang/pt_BR.json]: sorry, it was a writing error, but I already adjusted

File: lang/pt_BR.json (hunk 1 above)
Comment (vagnercruz) [reply lang/pt_BR.json]: sorry, it was a writing error, but I already adjusted

File: lang/pt_BR.json (hunk 3 above)
Comment (vagnercruz) [reply lang/pt_BR.json:315]: this is correct

**Review by vagnercruz — COMMENTED** [review]
finished

**Review by vagnercruz — COMMENTED** [review]
finished

**Review by wescopeland — APPROVED** [review]

**Pull request merged**